Dockerfile

```diff
@@ -125,7 +125,7 @@ RUN apt-get update && \
 ARG CUDA_SUPPORT
 ENV CUDA_SUPPORT=${CUDA_SUPPORT}
 RUN if [ "$CUDA_SUPPORT" = "yes" ]; then \
-        /tmp/components/cuda/install.sh; \
+        /tmp/components/cuda/install-cuda-10-0.sh; \
```

Comment:
Why do we need the original install.sh file? Why not just install.sh instead of install-cuda-10-0.sh?

Reply:
@nmanovic I agree I prefer the way you suggest it. In fact that is they way I have it locally.